### PR TITLE
modifying the multipath_test.py script for scenario testing

### PR DESCRIPTION
**Pull request by Naresh-ibm**
converted the code into functions, based on different scenarios,
So that each scenario can be ran when needed.
script also works fine as a whole.

Signed-off-by: Naresh Bannoth <nbannoth@in.ibm.com>

### CODE REVIEW

**Comment by narasimhan-v** [comment]
@Naresh-ibm job.log please

File: io/disk/multipath_test.py [hunk 1]
```diff
@@ -96,7 +96,7 @@ def setUp(self):
         for wwid in self.wwids:
             if wwid not in process.system_output('multipath -ll',
                                                  ignore_status=True,
-                                                 shell=True).decode("utf-8"):
+                                                 shell=True):
```
Comment (narasimhan-v) [inline io/disk/multipath_test.py]: why is the decode removed ?

File: io/disk/multipath_test.py [hunk 2]
```diff
@@ -151,46 +151,161 @@ def test(self):
...
+        err_paths = []
+        self.log.info("Removing and adding back all paths dynamically")
+        for wwid in self.wwids:
+            paths = multipath.get_paths(wwid)
+            for path in paths:
+                if multipath.remove_path(path) is False:
+                    self.log.info("couldn't remove in remove path: %s", path)
+                    err_paths.append(path)
+
+            time.sleep(180)
```
Comment (narasimhan-v) [inline io/disk/multipath_test.py]: can this 180 be initialised as a variable and used everywhere else ?

File: io/disk/multipath_test.py [hunk 3]
```diff
@@ -151,46 +151,161 @@ def test(self):
...
 
-        # Print errors
-        if msg:
-            self.fail("Some tests failed. Find details below:\n%s" % msg)
+    def test_suspend_resume_individual_mpath(self):
+        '''
+        suspending the mpathX and Resume it Back
+        '''
+        err_mpaths = []
+        time.sleep(10)
```
Comment (narasimhan-v) [inline io/disk/multipath_test.py]: why sleep here ?

File: io/disk/multipath_test.py [hunk 4]
```diff
@@ -151,46 +151,161 @@ def test(self):
...
+                err_mpaths.append(mpath)
+        if err_mpaths:
+            self.fail("error mpaths in suspnd indvdl mpaths: %s", err_mpaths)
+
+    def test_suspend_resume_all_mpath(self):
+        '''
+        suspending all the mpathX and then Resume it Back at once
+        '''
+        err_mpaths = []
+        time.sleep(10)
```
Comment (narasimhan-v) [inline io/disk/multipath_test.py]: why sleep here ?

File: io/disk/multipath_test.py [hunk 5]
```diff
@@ -151,46 +151,161 @@ def test(self):
...
+                err_mpaths.append(mpath)
+        if err_mpaths:
+            self.fail("error mpaths in suspnd all mpaths: %s" % err_mpaths)
+
+    def remove_add_mpath(self):
+        '''
+        Removing the mpathX and Add it Back
+        '''
+        err_mpaths = []
+        time.sleep(180)
```
Comment (narasimhan-v) [inline io/disk/multipath_test.py]: why sleep here ?

**Comment by Naresh-ibm** [comment]
closing this in favor of #1432

**Pull request closed without merging**